src/main/java/com/smatechnologies/opcon/restapiclient/model/dailyjob/details/IncidentTicketDetails.java

```diff
@@ -0,0 +1,39 @@
+package com.smatechnologies.opcon.restapiclient.model.dailyjob.details;
+
+public class IncidentTicketDetails {
+	
+	   public static final String RESOURCE = "incidentTickets";
+
+	    public static final String PROPERTY_ID = "id";
+	    public static final String TICKET_ID = "ticketId";
+	    public static final String TICKET_URL = "ticketUrl";
+
+	    private String id;
+	    private String ticketId;
+	    private String ticketUrl;
+		
+	    public String getId() {
+			return id;
```

Comment:
Please utilise formatter on all your files:
- Tabs are forbidden and must be spaces
- A lot of useless duplicated empty lines